### ISSUE TITLE
Staging Review finding: Duplicate headings in Review and Submit - Non-VA treatment records

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#117631]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Disability Benefits
**Product:** 21-526ez Disability Benefits Application
**Feature:** Update Form 4142 within 526 to the latest version

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility), @kkitagawa-bosch (content)

### Description

On the Review and Submit page, the heading text "Non-VA treatment records" is repeated mulitple times (3x). In this context, with clear sections of information, the duplicate headings don't clearly reference the content below each heading, which may be confusing for some users. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="706" height="853" alt="Multiple duplicate headings" src="https://github.com/user-attachments/assets/5963e690-8636-4964-b522-fb001fd407e9" />

</details>

## Recommended action

Provide headings that are specific to the content they precede, ensuring the headings accurately describe the following content/section. 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 2.4.6 Headings and Labels - Level AA](https://www.w3.org/TR/WCAG22/#headings-and-labels)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by comaurice** [comment]
Design ticket added as sub of this ticket

**Comment by adlco** [comment]
Hey @andaleliz , https://github.com/department-of-veterans-affairs/va.gov-team/issues/122457 is actually the ticket we planned into sprint 21

**Comment by andaleliz** [comment]
Oh got it - I must've been looking at the wrong view when I left that comment. my bad!
